pyproject.toml

```diff
@@ -1,3 +1,125 @@
+[build-system]
+requires = ["setuptools",  "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "mne-bids"
+description = "MNE-BIDS: Organizing MEG, EEG, and iEEG data according to the BIDS specification and facilitating their analysis with MNE-Python"
+dynamic = ["version"]
+authors = [
+    { name = "MNE-BIDS Developers" },
+]
+maintainers = [
+    { name = "Stefan Appelhoff", email = "stefan.appelhoff@mailbox.org" },
+]
+license = { text = "BSD-3-Clause" }
+readme = { file = "README.md", content-type = "text/markdown" }
+requires-python = ">=3.9"
+keywords = [
+    "meg",
+    "eeg",
+    "ieeg",
+    "bids",
+    "brain imaging data structure",
+    "neuroscience",
+    "neuroimaging",
+]
+classifiers = [
+    "Topic :: Scientific/Engineering",
+    "Intended Audience :: Science/Research",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved",
+    "Topic :: Software Development",
+    "Topic :: Scientific/Engineering",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX :: Linux",
+    "Operating System :: MacOS",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+]
+scripts = { mne_bids = "mne_bids.commands.run:main" }
+dependencies = [
+    "mne>=1.5",
+    "numpy>=1.21.2",
+    "scipy>=1.7.1",
+]
+[project.optional-dependencies]
+# Variants with dependencies that will get installed on top of those listed unter
+# project.dependencies
+
+# Dependencies for running the test infrastructure
+test = [
+    "mne>=1.5",
+    "numpy>=1.21.2",
+    "scipy>=1.7.1",
```

Comment:
No need to repeat the deps here. Everything listed here will be installed **in addition** to the "core" reps.

```suggestion
```